polaris/_mixins.py

```diff
@@ -69,3 +71,17 @@ def verify_checksum(self, md5sum: str | None = None):
             raise PolarisChecksumError(
                 f"The specified checksum {md5sum} does not match the computed checksum {self.md5sum}"
             )
+
+
+class FormattingMixin:
```

Comment:
**Suggestion**: Now that we have this, let's use it for the `ProgressIndicator` as well!